apps/confidential/confidential.go

```diff
@@ -243,97 +243,74 @@ func AutoDetectRegion() string {
 // package doc. A new Client should be created PER SERVICE USER.
 // For more information, visit https://docs.microsoft.com/azure/active-directory/develop/msal-client-applications
 type Client struct {
-	base base.Client
-
-	cred *accesstokens.Credential
-
+	accessor                          cache.ExportReplace
```

Comment:
should we call it more in the lines of `cache serialization` ? Maybe `cacheSerializer` ?

Reply:
(not sure if this is part of public API. If not, ignore comment)

Reply:
Just FYI, names beginning with a lowercase letter are private